card/mod/machines/lib/javascript/decko_filter.js.coffee

```diff
@@ -66,7 +66,11 @@ updateSelectedCount = (el) ->
   count = selectedBin(el).children().length
   filterBox(el).find("._selected-items").html count
   deselectAllLink(el).attr "disabled", count == 0
-  addSelectedButton(el).attr "disabled", count == 0
+  if count > 0
+    addSelectedButton(el).removeClass("disabled")
+  else
+    addSelectedButton(el).addClass("disabled")
+
```

Comment:
is it correct that the button needed new handling but the deselectLink didn't?

Reply:
I don't know why an href attribute on a button worked before but it doesn't anymore so I changed the button to a link.